assets/sass/_mixins.scss

```diff
@@ -273,17 +273,17 @@
   margin: auto;
 
   @if $padding {
-    padding-left: 7%;
-    padding-right: 7%;
+    padding-left: #{$site-spacing}px;
```

Comment:
Should we be using `rem` as we do in the media query?

Reply:
I am following how the rest of the spacing is in the variables stylesheet